.github/CODEOWNERS

```diff
@@ -15,19 +15,31 @@
 
 # DocOps Writers
 *                           @nginx/nginx-docs
+
 # DocOps Engineers
 .github/*                   @nginx/docs-engineering
-# NGINX Plus
-content/nginx/*             @nginx/plus-docs-approvers
+
 # NGINX Agent
 content/nginx/nms/agent/*   @nginx/agent-docs-approvers
```

Comment:
Agent, NIC, and NGF teams already exist in the org; can we leverage those instead of creating new groups?

Reply:
```suggestion
content/nginx/nms/agent/*   @nginx/nginx-agent 
```

Reply:
Appropriate permissions have been granted, so this should be good to go.